data_steward/test/unit_test/vocabulary_test.py

```diff
@@ -0,0 +1,97 @@
+import unittest
```

Comment:
According to [pep 8],(https://www.python.org/dev/peps/pep-0008/#imports) imports should be grouped as, Python Standard Imports, Third party imports, and Project imports.  This should be changed to:

```
import os
import StringIO
import unittest

import mock

import test_util
import vocabulary
```

Also, Pep 8 discourages importing via `*` and importing a function who is considered private.  If it's for testing purposes, the functions that use `_transform_csv` should handle this information and checks.  If `_transform_csv` may be used elsewhere, rename it and drop the underscore.